tests/data_access/models/test_todo_entry_model.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+import src.data_access.models.user_model as user_model
+from src.data_access.models.todo_entry import TodoEntry
+
+
+class TestSerialize:
+    def test_serialize_calls_to_nested_users_serialize(self, mocker):
+        mock_user = mocker.patch.object(user_model, "User")
+        mock_user.serialize.return_value = {}
+        expected_serialized_todo = {
+            "title": "title",
+            "id": None,
+            "date_time": datetime.now(),
+            "user": {},
+            "user_id": 1,
+        }
+        todo = TodoEntry(
```

Comment:
mock the object (if u have bug in the constructor it will affect this test)

Reply:
how am I supposed to both mock TodoEntry and also test its method at the same time? 
if I have a bug in the ctor then this test would fail which is good